app/views/comments/_form.html.erb

```diff
@@ -1,5 +1,5 @@
-<li class="list-group-item">
-  <%= form_with(model: comment) do |form| %>
+<li id="<%= dom_id(comment.photo) %>_<%= dom_id(comment) %>_form" class="list-group-item">
+  <%= form_with(model: comment, local: false) do |form| %>
```

Comment:
great job for being mindful about when to use local:false and when to use remote:true